src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -76,9 +87,11 @@ export const styles = StyleSheet.create({
   },
   groupName: {
     fontSize: normalizeFontSize(24),
-    fontFamily: fonts.inter[500],
     lineHeight: normalizeFontSize(29.05),
-    color: '#000'
+    color: '#000',
+    width: '100%',
+    paddingHorizontal: dimen.normalizeDimen(20),
+    fontWeight: 'bold'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `groupName` style has been updated to include padding and font weight. Ensure that these changes do not affect the layout in an undesired way.

```diff
-   groupName: {
-     fontSize: normalizeFontSize(24),
-    fontFamily: fonts.inter[500],
-     lineHeight: normalizeFontSize(29.05),
-    color: '#000'
-   },
+   groupName: {
+     fontSize: normalizeFontSize(24),
+     lineHeight: normalizeFontSize(29.05),
+    color: '#000',
+    width: '100%',
+    paddingHorizontal: dimen.normalizeDimen(20),
+    fontWeight: 'bold'
+  },
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->